### PR TITLE
6 distinct clonal families

### PR DESCRIPTION
**Pull request by metasoarous**

### CODE REVIEW

File: src/components/explorer/visualization.js [hunk 1]
```diff
@@ -24,6 +29,10 @@ const MyVegaLite = args => {
   return <VegaLite {...args}/>
 }
 
+
```
Comment (eharkins) [inline src/components/explorer/visualization.js]: Thanks for all these comments, been meaning to clean up this file.

**Comment by metasoarous** [comment]
@eharkins Creating a PR to hash the rest of this out and have a complete diff to look at.

Some things left to do:

* right now it's just selecting a random tree; need to have a sane default mechanism here in the selector machinery
* add UI & MVC lofic for switching between trees

I'm a little tempted to merge this in as is and open separate issues for the above, since it's working and since I'd rather keep the branches from diverging much more (I merged master in here, so it would be easy to merge now, but will get less so the more other commits pile on top of master in the interim). What do you think @eharkins?

File: src/selectors/selectedFamily.js [hunk 2]
```diff
@@ -1,12 +1,33 @@
 import { createSelector, createSelectorCreator, defaultMemoize } from 'reselect';
 import * as _ from 'lodash';
 
-const getSelectedFamily = (state) => {
+// The naming structure here needs to be cleaned up;
```
Comment (eharkins) [inline src/selectors/selectedFamily.js]: Just in terms of the selectors or the whole redux store naming conventions?

Comment (metasoarous) [reply src/selectors/selectedFamily.js]: The selector names; the one for the tree structure especially, since it's not just a tips collection.

**Pull request merged**